added labels input to save_url method

naveennamani opened the pull request:
## 📑 Description
Now, labels can be added when saving a url by calling with `save_url(url, ["label1", "label2"])`.

## ✅ Checks
- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information

naveennamani committed:
added labels input to save_url method

Now, labels can be added when saving a url by calling with `save_url(url, ["label1", "label2"]`.

yazdipour commented:
Hey Naveen, Thanks for your contribution. You need to update the unittest to match your changes before PR.

naveennamani committed:
fixing sourcery suggestion

naveennamani committed:
added unittest case `test_save_url_with_labels`

naveennamani committed:
fixed error in passing variable_values

naveennamani commented:
Dear @yazdipour, I've added a unittest test case `test_save_url_with_labels`. Also, I fixed an error in passing the input values, and all the test cases are now passing.
